### DISCUSSION TITLE
Video Download Failed

### DISCUSSION THREAD

**Discussion by UnLiViNgHuMaN** [aclap-dev/video-downloadhelper#607] [Help and Q&A]
### Your question

I have downloaded other movies successfully from this website but this movie failed. Tried on two different video files and same results.

### Addon & CoApp version and details

{
  "version": "9.0.1.18",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://movieshd.watch/watch-movie/watch-the-fall-guy-online-108544.10431826",
  "error": {
    "details": "SideDownload error: : [tls @ 0000023baac1e500] error:0A00010B:SSL routines::wrong version number\r\n[tls @ 0000023baac04400] error:0A00010B:SSL routines::wrong version number\r\n    Last message repeated 2 times\r\n[tls @ 0000023baac169c0] error:0A00010B:SSL routines::wrong version number\r\n    Last message repeated 1 times\r\n[hls @ 0000023baab7b7c0] Error when loading first segment 'https://bnv.spreadsheetss.com/_v11/93220242079a12bf6594a3f0d25c7198c2b06caa97fe325d0f2c2561f6b4a90a879816d9a90cd66215016a5b0c60825d6926a32cb76ee329e621114f4c18ec668c2bede8e0de9ebb757edd4953317af47ecdaa28d060691216627b0297f403b2a553314e4b7424b76ee0d4bf616fb5f3c26b0f55220c99c33f252a5f176d2cc7/1080/seg-1-v1-a1.jpg'\r\n[in#0 @ 0000023baab7b600] Error opening input: Invalid data found when processing input\r\nError opening input file https://l.pollllop.com/_v11/93220242079a12bf6594a3f0d25c7198c2b06caa97fe325d0f2c2561f6b4a90a879816d9a90cd66215016a5b0c60825d6926a32cb76ee329e621114f4c18ec668c2bede8e0de9ebb757edd4953317af47ecdaa28d060691216627b0297f403b2a553314e4b7424b76ee0d4bf616fb5f34cee9d744dba8e0e6fddb92f946bb8a6/1080/index.m3u8.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_7165609612458878",
    "error": "coapp_failure",
    "id": "coapp_failure_b85f801a-cccc-44fa-9eac-c1f326aad404"
  }
}

{
  "version": "9.0.1.18",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://movieshd.watch/watch-movie/watch-the-fall-guy-online-108544.10431826",
  "error": {
    "details": "SideDownload error: : [tls @ 000001c6f7f7e500] error:0A00010B:SSL routines::wrong version number\r\n[tls @ 000001c6f7f64400] error:0A00010B:SSL routines::wrong version number\r\n    Last message repeated 2 times\r\n[tls @ 000001c6f7f769c0] error:0A00010B:SSL routines::wrong version number\r\n    Last message repeated 1 times\r\n[hls @ 000001c6f7edb7c0] Error when loading first segment 'https://bnv.spreadsheetss.com/_v11/93220242079a12bf6594a3f0d25c7198c2b06caa97fe325d0f2c2561f6b4a90a25c9f94d4c1f2e42b1ffa7dc829b646c18e5025228ab2397f9d4786675c0a1487b368e1f94e292a1029cafa274d6927f9b4a3cf561bccac95f88758807f9e3048dfe0b8fdbaecf0c2169aa98bf07eff422405c74c8317c5b268df1c0ce3d21d6/720/seg-1-v1-a1.jpg'\r\n[in#0 @ 000001c6f7edb600] Error opening input: Invalid data found when processing input\r\nError opening input file https://l.pollllop.com/_v11/93220242079a12bf6594a3f0d25c7198c2b06caa97fe325d0f2c2561f6b4a90a879816d9a90cd66215016a5b0c60825d6926a32cb76ee329e621114f4c18ec668c2bede8e0de9ebb757edd4953317af47ecdaa28d060691216627b0297f403b2a553314e4b7424b76ee0d4bf616fb5f34cee9d744dba8e0e6fddb92f946bb8a6/720/index.m3u8.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2101336518724197",
    "error": "coapp_failure",
    "id": "coapp_failure_1cd82291-4689-4571-8026-be0be92f6e87"
  }
}

**Comment by paulrouget** [comment]
Thank you for reporting the issue! I'll look into it.

Note: ff-failure, most likely: `SSL routines::wrong version number`.
